lib/dash/dash_parser.js

```diff
@@ -648,6 +648,11 @@ shaka.dash.DashParser = class {
     // best latency
     if (this.lowLatencyMode_) {
       minBufferTime = 0;
+      const defaultPresentationDelay =
+          this.config_.defaultPresentationDelay || minBufferTime * 1.5;
+      const presentationDelay = suggestedPresentationDelay != null ?
+          suggestedPresentationDelay : defaultPresentationDelay;
+      presentationTimeline.setDelay(presentationDelay);
```

Comment:
I think instead of changing the delay, we could just move the code that sets `minBufferTime` to 0 up to line 499, so it will happen before the `PresentationTimeline` is created.